src/renderer/pages/Basket/model/basket-page-model.ts

```diff
@@ -57,13 +130,58 @@ sample({
   target: $selectedTxs,
 });
 
+sample({
+  clock: [$basketTransactions, networkModel.$apis],
+  source: {
+    transactions: $basketTransactions,
+    apis: networkModel.$apis,
+  },
+  filter: ({ transactions, apis }) => {
+    const chains = new Set(transactions.map((t) => t.coreTx.chainId));
+
+    return [...chains].some((chainId) => apis[chainId]);
+  },
+  fn: ({ transactions }) => {
+    return transactions;
+  },
+  target: validateFx,
+});
+
+sample({
+  clock: [
+    transferValidateModel.events.txValidated,
+    addProxyValidateModel.events.txValidated,
+    addPureProxiedValidateModel.events.txValidated,
+    removeProxyValidateModel.events.txValidated,
+    removePureProxiedValidateModel.events.txValidated,
+    bondNominateValidateModel.events.txValidated,
+    nominateValidateModel.events.txValidated,
+    bondExtraValidateModel.events.txValidated,
+    restakeValidateModel.events.txValidated,
+    unstakeValidateModel.events.txValidated,
+    withdrawValidateModel.events.txValidated,
+  ],
+  source: $invalidTxs,
+  fn: (txs, { id, result }) => {
+    if (!result) {
+      txs.delete(id);
+    } else {
+      txs.add(id);
+    }
+
+    return txs;
+  },
+
+  target: $invalidTxs,
```

Comment:
Are you sure this is triggered? Because we alter `txs` in place and not return new reference to `Set`